### PR TITLE
Adding voucher options to ACF and posting to calais

### PR DESCRIPTION
**Pull request by garethfoote**
Can now specify a voucher within a promo. 
Voucher data is being sent to calais.

### CODE REVIEW

**Comment by ecoad** [comment]
Any new ACF keys can you give them proper names please? I see there are some existing keys which are in there which are autogenerated but we'll have to live with that sin.

**Comment by garethfoote** [comment]
@ecoad Updated PR.

File: app/custom-fields/promo.php [hunk 1]
```diff
@@ -16,6 +16,21 @@
   'key' => 'promotion_data_capture',
   'title' => 'Promotion',
   'fields' => array (
+		array (
+			'key' => 'field_564335a4d8814',
```
Comment (ecoad) [inline app/custom-fields/promo.php]: @garethfoote Can you rename this autogenerated field key please?

Comment (ecoad) [inline app/custom-fields/promo.php]: Actually I'll do it here. No worries

**Comment by jonsherrard** [comment]
Sorry got distracted, had checked. All looks good :+1:

**Pull request merged**